src/bootstrap/builder.rs

```diff
@@ -328,10 +335,12 @@ impl StepDescription {
             );
         }
 
+        let mut ran = false;
         if paths.is_empty() || builder.config.include_default_paths {
             for (desc, should_run) in v.iter().zip(&should_runs) {
                 if desc.default && should_run.is_really_default() {
-                    desc.maybe_run(builder, should_run.paths.iter().cloned().collect());
+                    ran =
```

Comment:
`ran ||=` is a bit nicer here.

Reply:
Hmm, does that work? I can't seem to get it to work.

Reply:
Sorry, single | -- `ran |= ...`